### DISCUSSION TITLE
Displaying (Wrong hours) ago

### DISCUSSION THREAD

**Discussion by vlrfsg** [community/community#10567] [Mobile] [answered]
I found this issue on Android today.
Within 15 minutes after I commited and pushed something, the application said that had been 9 hours ago. Since I'm using UTC+9 time, this seems something about timezones. Comments are also in this case.

I'm afraid that this late post will bother the GitHub Mobile team if they are going to fix this in the next release. At the same time, I'm also a bit curious how it is displayed in west of the Atlantic Ocean.

**Comment by brianlovin** [comment, marked as answer]
Hey, this issue should be fixed in the next version of the app — could you double check that you're on the latest version already, and it might be the case that our next build will ship soon and should fix the bug. Thanks for taking the time to report 😊

**Reply by vlrfsg** [reply]
Thank you. The version 1.49.2 works properly. Sorry for no version info attached.

**Reply by brianlovin** [reply]
Glad it's working, thanks for following up.
